src/provider.py

```diff
@@ -94,6 +99,7 @@ def relation_config(
             "password": password,
             "chroot": chroot,
             "acl": acl,
+            "acls-added": str(acls_added),
```

Comment:
**niptick** maybe str(...) is not really needed, given it is just set above to either "true" or "false"

Reply:
Good catch, also a left-over from initial when they were originally boolean. Thanks!!